src/components/ClientForm/ClientForm.module.scss

```diff
@@ -0,0 +1,63 @@
+.main {
+  background-color: var(--light-brown);
+  border-radius: 25px;
+  width: 793px;
+  margin-top: 36px;
+  padding: 36px;
+
+  @media (max-width: 1022px) {
+    // width: calc(100vw - 24px);
+    width: 100vw;
+  }
+}
+
+.formHeader {
+  font-size: 32px;
+  margin-bottom: 24px;
+}
+
+.inputFieldContainer {
+  display: flex;
+  flex-direction: column;
+  width: 319px;
```

Comment:
if we move the max-width to wrapping the entire form/h1, the inputs can just stretch fully to fill